cloudify/cluster.py

```diff
@@ -55,7 +55,11 @@ def do_request(self, *args, **kwargs):
             try:
                 return super(ClusterHTTPClient, self).do_request(*args,
                                                                  **kwargs)
-            except (requests.exceptions.ConnectionError):
+            except (requests.exceptions.ConnectionError) as error:
+                self.logger.debug(
+                    'Connection error when trying to connect to '
+                    'manager {0}'.format(error.message)
```

Comment:
just `error`

using `.message` is wrong and would fail CI if this was tested

Reply:
will update that